src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml

```diff
@@ -211,10 +210,54 @@
+                        <Style TargetType="{x:Type Button}">
+                            <Setter Property="Template">
+                                <Setter.Value>
+                                    <ControlTemplate TargetType="{x:Type Button}">
+                                        <Grid>
+                                            <Border x:Name="DotsBackgroundBorder"
+                                                Width="24"
+                                                Height="24"
+                                                Background="Transparent"
+                                                CornerRadius="2" />
+                                            <Image x:Name="DotsImage"
+                                               Width="16px"
+                                               Height="16px"
+                                               Margin="1.5,0,0,0"
+                                               HorizontalAlignment="Center"
+                                               VerticalAlignment="Center"
+                                               Stretch="UniformToFill" />
+                                        </Grid>
+                                        <ControlTemplate.Triggers>
+                                            <Trigger Property="IsMouseOver" Value="True">
+                                                <Setter TargetName="DotsBackgroundBorder" Property="Background" Value="{StaticResource NodeOptionsButtonBackground}" />
+                                                <Setter TargetName="DotsImage" Property="Source" Value="/DynamoCoreWpf;component/UI/Images/sort-hover-16px.png" />
+                                            </Trigger>
+                                            <Trigger Property="IsMouseOver" Value="False">
+                                                <Setter TargetName="DotsBackgroundBorder" Property="Background" Value="Transparent" />
+                                                <Setter TargetName="DotsImage" Property="Source" Value="/DynamoCoreWpf;component/UI/Images/Sort.png" />
+                                            </Trigger>
+                                        </ControlTemplate.Triggers>
+                                    </ControlTemplate>
+                                </Setter.Value>
+                            </Setter>
+                        </Style>
+                    </Button.Style>
+                </Button>
+                <Grid.ContextMenu>
+                    <ContextMenu Name="SuggestionsContextMenu" Style="{StaticResource ContextMenuStyle}">
+                        <MenuItem Header="{x:Static p:Resources.MachineLearning}" x:Name="miMLRecommendation" Click="onSuggestion_Click" IsCheckable="True" IsChecked="{Binding IsML}"/>
+                        <MenuItem Header="{x:Static p:Resources.ObjectType}" x:Name="miObjectType" Click="onSuggestion_Click" IsCheckable="True" IsChecked="{Binding IsObjectType}"/>
```

Comment:
There should be converter to flip the bool value, I would love to keep a single boolean for these two options

Reply:
Done.